### ISSUE TITLE
tensor size error using Real ESRGAN with blend tab

### ISSUE THREAD

**Issue by ssube** [ssube/onnx-web#149]
```
        Tile 1/1                                                                                                                                                                                      [2023-02-15 04:41:57,940] WARNING: onnx_web.server.device_pool: job upscale_311954741_58882e67b0aeb875055748e5d2660d73954cbedb79998bac08264cdb0e85f3ff_1676436117.png failed with an error: ['Traceback (most recent call last):
', '  File "/home/ssube/onnx-web/api/onnx_web/server/device_pool.py", line 243, in job_done
    f.result()
', '  File "/usr/lib/python3.10/concurrent/futures/_base.py",
 line 451, in result
    return self.__get_result()
', '  File "/usr/lib/python3.10/concurrent/futures/_base.py", line 403, in __get_result
    raise self._exception
', '  File "/usr/lib/python3
.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
', '  File "/home/ssube/onnx-web/api/onnx_web/diffusion/run.py", line 262, in run_blend_pipeline
    image = run_upscale_correction(
', '  File "/home/ssube/onnx-web/api/onnx_web/server/upscale.py", line 61, in run_upscale_correction
    return chain(
', '  File "/home/ssube/onnx-web/api/on
nx_web/chain/base.py", line 147, in __call__
    image = stage_pipe(
', '  File "/home/ssube/onnx-web/api/onnx_web/chain/upscale_resrgan.py", line 112, in upscale_resrgan
    output, _ = upsampler.enhance(output, outscale=upscale.outscale)
', '  File "/home/ssube/onnx-web/api/onnx_env/lib/python3.10/site-packages/torch/autograd/grad_mode.py", line 27, in decorate_context
    return func(*
args, **kwargs)
', '  File "/home/ssube/onnx-web/api/onnx_env/lib/python3.10/site-packages/realesrgan/utils.py", line 221, in enhance
    self.tile_process()
', '  File "/home/ssube/onnx-web/api/onnx_env/lib/python3.10/site-packages/realesrgan/utils.py", line 178, in tile_process
    self.output[:, :, output_start_y:output_end_y,
', 'RuntimeError: The expanded size of the tensor (2048) mu
st match the existing size (1024) at non-singleton dimension 3.  Target sizes: [1, 3, 2048, 2048].  Tensor sizes: [3, 1024, 1024]
']
[2023-02-15 04:41:57,940] DEBUG: onnx_web.utils: running garbage collection                                                                                                                           
[2023-02-15 04:42:02,736] DEBUG: onnx_web.server.device_pool: pruning 1 of 1 pending jobs                                                                                                             [2023-02-15 04:42:02,736] DEBUG: onnx_web.server.device_pool: pruning 1 of 11 recent jobs  
```

**Comment by ssube** [comment]
The fix for #148 seems to have had some impact on this, but it's still not working:

```
[2023-02-15 23:53:03,332] INFO: onnx_web.chain.upscale_resrgan: upscaling image with Real ESRGAN: x4                                                                                
[2023-02-15 23:53:03,333] DEBUG: onnx_web.server.device_pool: job upscale_554257420_d7d4b693513c8f03475b89cdd48c5fa8ab5f223b0c73994f49de44928776ae17_1676505183.png assigned to devi
ce cuda - CUDAExecutionProvider ({'device_id': 0})                                                                                                                                  
[2023-02-15 23:53:03,339] INFO: werkzeug: 10.2.2.16 - - [15/Feb/2023 23:53:03] "GET /api/ready?output=upscale_554257420_d7d4b693513c8f03475b89cdd48c5fa8ab5f223b0c73994f49de44928776
ae17_1676505183.png HTTP/1.1" 200 -                                                                                                                                                 
EP Error using [('CUDAExecutionProvider', {'device_id': 0})]                                                                                                                        Falling back to ['CUDAExecutionProvider', 'CPUExecutionProvider'] and retrying.
[2023-02-15 23:53:03,719] DEBUG: onnx_web.chain.upscale_resrgan: loading Real ESRGAN upscale model from ../models/upscaling-real-esrgan-x4-plus.onnx                                [2023-02-15 23:53:03,720] INFO: werkzeug: 10.2.2.16 - - [15/Feb/2023 23:53:03] "GET /output/txt2img_1813645624_483221535776f7d9c02c2eb2bb8d98178124af7ddfe545f29c3c9648c79ccba6_1676
501656.png HTTP/1.1" 404 -                                                                
[2023-02-15 23:53:03,720] INFO: werkzeug: 10.2.2.16 - - [15/Feb/2023 23:53:03] "GET /output/txt2img_672086775_2d4db2a48225a2f9ce829bb1a4ccf1e0673337ac2a1cd4043346c86e783fa033_1676504772.png HTTP/1.1" 304 -          
[2023-02-15 23:53:03,721] INFO: werkzeug: 10.2.2.16 - - [15/Feb/2023 23:53:03] "GET /output/txt2img_200416719_e5df6f59d03bf4bb7663f527c0b277ce9a38b252bf23f3b3ae9ad08de8ea66eb_1676503563.png HTTP/1.1" 304 -
[2023-02-15 23:53:03,786] DEBUG: onnx_web.server.model_cache: Cache limit set to 0, not caching model: resrgan
[2023-02-15 23:53:03,787] DEBUG: onnx_web.utils: running garbage collection
[2023-02-15 23:53:08,450] INFO: werkzeug: 10.2.2.16 - - [15/Feb/2023 23:53:08] "GET /api/ready?output=upscale_554257420_d7d4b693513c8f03475b89cdd48c5fa8ab5f223b0c73994f49de44928776
ae17_1676505183.png HTTP/1.1" 200 -
[2023-02-15 23:53:13,463] INFO: werkzeug: 10.2.2.16 - - [15/Feb/2023 23:53:13] "GET /api/ready?output=upscale_554257420_d7d4b693513c8f03475b89cdd48c5fa8ab5f223b0c73994f49de44928776
ae17_1676505183.png HTTP/1.1" 200 -
[2023-02-15 23:53:18,480] INFO: werkzeug: 10.2.2.16 - - [15/Feb/2023 23:53:18] "GET /api/ready?output=upscale_554257420_d7d4b693513c8f03475b89cdd48c5fa8ab5f223b0c73994f49de44928776
ae17_1676505183.png HTTP/1.1" 200 -
        Tile 1/1
        Tile 1/1
[2023-02-15 23:53:21,370] INFO: onnx_web.chain.upscale_resrgan: final output image size: 2048x2048
[2023-02-15 23:53:21,452] INFO: onnx_web.chain.base: finished stage upscale_resrgan, result size: 2048x2048
[2023-02-15 23:53:21,453] INFO: onnx_web.chain.base: finished pipeline in 0:00:18.120717, result size: 2048x2048
```

produces

![image](https://user-images.githubusercontent.com/451959/219222113-c9cc4bfc-9b46-4223-b75e-0ff86fd350c7.png)

**Comment by ssube** [comment]
I think this was a combination of #140 and #148, because with both of those fixes, I can no longer reproduce with any of the upscaling models.
